service/src/test/java/com.codeforcommunity/processor/ProcessorImplTest.java

```diff
@@ -19,4 +25,12 @@ void setup() {
   public void testSomethingTest() {
     assertTrue(true);
   }
+
+  @Test
+  public void testFetchExists() {
+    DSLContext db = mockDb.getContext();
+    mockDb.addExistsReturn();
+    db.fetchExists(db.selectFrom(Tables.USERS).where(Tables.USERS.ID.eq(1)));
+    assertEquals(1, mockDb.getSqlOperationBindings().get(OperationType.EXISTS).size());
+  }
```

Comment:
Let me know if this test works for you. I'm getting an exception about reflection issues whenever I try to run `JooqMock` (even on that didn't happen before) on Java 8.

Reply:
Will be fixed in another PR